OpenAI/OpenAI_Generate_Text_to_Speech.ipynb

```diff
@@ -0,0 +1,262 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1fe9f56e-561c-4f52-aef8-b861c9462107",
+   "metadata": {},
+   "source": [
+    "**Author:** [Sriniketh Jayasendil](http://linkedin.com/in/sriniketh-jayasendil/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "31ea7cdb-e10d-43fc-b026-f69249a59736",
+   "metadata": {},
+   "source": [
+    "**Description:** This notebook focusses on using OpenAI for text-to-speech generation using [gTTS](https://pypi.org/project/gTTS/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "distinguished-truth",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "## Input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "numeric-mediterranean",
+   "metadata": {},
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "potential-surfing",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import naas\n",
+    "\n",
+    "# OpenAI\n",
+    "try:\n",
+    "    import openai\n",
+    "except ModuleNotFoundError:\n",
+    "    !pip install openai\n",
+    "    import openai\n",
+    "\n",
+    "# Google Text to Speech API\n",
+    "try:\n",
+    "    from gtts import gTTS\n",
+    "except ModuleNotFoundError:\n",
+    "    !pip install gTTS\n",
+    "    from gtts import gTTS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aggressive-trustee",
+   "metadata": {},
+   "source": [
+    "### Setup Variables"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "750610b0-f924-4084-98b3-a0852d1acbd0",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# naas.secret.add(name=\"OPENAI_API_KEY\", secret=\"<REPLACE_WITH_YOUR_KEY>\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "continuous-melbourne",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "#inputs\n",
+    "openai.api_key = naas.secret.get(name=\"OPENAI_API_KEY\")\n",
+    "language = 'en'\n",
+    "prompt = input('Enter the prompt: ')\n",
+    "\n",
+    "#outputs\n",
+    "speech_output = ''"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "registered-showcase",
+   "metadata": {},
+   "source": [
+    "## Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "tested-astrology",
+   "metadata": {},
+   "source": [
+    "### Establish connection with OpenAI and get the response"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "crude-louisville",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "response = openai.Completion.create(\n",
+    "  model=\"text-davinci-003\",\n",
+    "  prompt=prompt,\n",
+    "  temperature=0,\n",
```

Comment:
I think good basic parameters would be the same as the playground:
Temperature: 0.7 (We are getting good results with 0.5 as well)
Top_P:1.0
max_tokens: 256 (I really feel like 64 is too small)